Library/Formula/unbound.rb

```diff
@@ -19,13 +19,10 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-libevent=#{Formula["libevent"].opt_prefix}",
                           "--with-ssl=#{Formula["openssl"].opt_prefix}"
+    inreplace "doc/example.conf", 'username: "unbound"', "username: \"#{ENV["USER"]}\""
```

Comment:
I've revised the commit to do so.

Works over multiple reinstalls when I'm testing locally. And it should play nice with installs from the previous versions of the formula: they won't contain the sentinel value in their config files, so they will be silently left alone, which is what we want.

I'll try testing it with the bottle once the test bot run finishes.